### PR TITLE
Logic/map geo

### PR DESCRIPTION
**Pull request by Hiperultimate**
Address input is now a choice box.
While registering accounts, the map now looks at address input and sets geolocation on typing and selecting through choice box. 
Implemented it for both patient and doctor registration. Unknown address results in LAT 0, LONG 0.

### CODE REVIEW

File: client/src/pages/home/home.css [hunk 1]
```diff
@@ -1,3 +1,28 @@
 .home-page{
     background-color: var(--green-secondary);
 }
+
+.adjust-doctor-list{
```
Comment (DamFlancoz) [inline client/src/pages/home/home.css:5]: Just doctor-list should suffice. Describing functionality names are better used if you reuse these classes/adjustments somewhere else too.

File: client/src/utils/filters/filterDoctors.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  location,
+  feeValue,
+  specializations,
+  treatments,
+) {
+    const doctorCount = doctorList.length;
+    const filteredList = [];
+
+    for(let i = 0 ; i < doctorCount; i++) {
+        const filterDoc = doctorList[i];
```
Comment (DamFlancoz) [inline client/src/utils/filters/filterDoctors.js:46]: Maybe it can used restructured into doctorList.filter(...) if it is provided.

**Review by DamFlancoz — CHANGES_REQUESTED** [review]
Good code, just added a couple of considerations.

File: client/src/pages/home/home.css (hunk 1 above)
Comment (Hiperultimate) [reply client/src/pages/home/home.css:5]: Oh I see

**Pull request merged**